apps/server/src/widgetEndpoint.ts

```diff
@@ -27,6 +27,7 @@ export const widgetHandler = (req: Request, res: Response) => {
     singleAccountSelect: Joi.bool(),
     userId: Joi.string().required(),
     token: Joi.string(),
+    aggregatorOverride: Joi.string().valid(...aggregators),
```

Comment:
I think in this case it'd be better to only allow the non-sandbox aggregators names like "mx", "sophtron", "akoya". Instead of allowing "mx_int", "akoya_sand" etc. This is because the institution list which elasticsearch uses only uses the non-test agg names. I think if you had a test where aggregatorOverride is "mx_int" it wouldn't return any institutions.